lib/index.js

```diff
@@ -49,14 +49,18 @@ $(document).ready(function() {
 
         canvas.addEventListener('mousedown', function(event) {
             var that = this
-            var mousePos = playerClub.getMousePosition(that, event)
-            moveBall(mousePos)
-            if(game.ball.inHole === false) {
-                var stroke = strokeCounter++
-                    domManipulator.insertStroke(stroke);
-                    // possobly store the stroke count in a player session
+            if (game.ball.moving){
```

Comment:
can we change this to just have a if statement and get rid of the else, like if (game.ball.moving === false) ?